app/assets/stylesheets/components/_message.pcss

```
@@ -0,0 +1,21 @@
+.message {
+  &__content {
+    @apply ml-5;
+  }
+
+  &__title {
+    @apply font-bold text-white text-sm tracking-wide;
+  }
+
+  &__item {
+    @apply text-white-600 text-xs tracking-wider;
+  }
+
+  &_bullet-icon {
```

Why not simply:

```suggestion
  &__icon {
```

Also there seems to be a missing underscore.

---

I changed to `list-icon` as it belongs to the list